tools/ncf.py

```diff
@@ -421,13 +423,20 @@ def check_technique_metadata(technique_metadata):
     if technique_metadata[key] == "":
       raise NcfError("A technique must have a "+key+", but there is none")
 
+  # File name of the technique will be invalid on most filesystem, do not save technique
+  if len(technique_metadata['bundle_name']) > 253:
+    raise NcfError("Bundle names longer than 255 characters won't work on most filesystems.")
+  elif len(technique['bundle_name']) > 100:
+      warnings.push("Bundle names longer than 100 characters may not work on some filesystems (Windows, in particular).")
```

Comment:
what happenswhen this warning appears ? Does the user need to confirm something ?
If not, technique is written, but is there a way for the user to change the lengrh of the technique filename afterward ?